logging/logkey/constants.go

```diff
@@ -23,6 +23,8 @@ const (
 	// Key is the key (namespace/name) being reconciled.
 	Key = "knative.dev/key"
 
+	TraceId = "knative.dev/traceid"
```

Comment:
Can you please add the documentation for this? Something like:

// TraceId is used to track an asynchronous or long running operation within Knative.

Reply:
ok, done